app/components/Resources/ResourcesTable.js

```diff
@@ -13,7 +13,13 @@ const cats = {
 class ResourcesTable extends Component {
 	constructor() {
 		super();
-		this.state = {categoryName: 'Category Name', resources: [], location: null};
+		this.state = {
+			categoryName: 'Category Name', 
```

Comment:
Ah, we fetch it in `loadResourcesFromServer()` below. I briefly talked to @achadha about react-redux a few weeks ago. We should look into whether that's the direction we want to go for the project; I think it leverages React's virtues well.